go.mod

```diff
@@ -147,36 +147,3 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
-
-replace (
```

Comment:
removing entire section is little bit scary

Reply:
I'm looking into a solution to avoid this. Replace was a hack when there were dependency issues. Now it should be ok, especially for k8s.io deps. I'd like to play a bit with these dependencies before continuing.